operator-framework-core/pom.xml

```diff
@@ -58,7 +58,6 @@
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
-      <scope>test</scope>
```

Comment:
I am not sure if it is right to include `assertj` as a dependency when build time rather than in a test environment.

Reply:
Yes, if this could be done nicer, would be great.

Reply:
possible alternative is to use https://github.com/java-diff-utils/java-diff-utils

Reply:
Let me know if I should remove the last commit or not.

Reply:
> possible alternative is to use https://github.com/java-diff-utils/java-diff-utils

This looks good to me, the library itself doesn't seem heavy.
It looks like there may be another opportunity to use this library for other purposes in the future.

Reply:
Yes, I was going to comment on using this lib instead of assertj. Thanks!